airflow/api/common/experimental/__init__.py

```diff
@@ -29,12 +29,10 @@ def check_and_get_dag(dag_id: str, task_id: Optional[str] = None) -> DagModel:
     if dag_model is None:
         raise DagNotFound("Dag id {} not found in DagModel".format(dag_id))
 
-    def read_store_serialized_dags():
-        from airflow.configuration import conf
-        return conf.getboolean('core', 'store_serialized_dags')
+    from airflow.configuration import conf
```

Comment:
Any reason this import is inside the function?

Reply:
Nope